tests/unit/utils/data.test.ts

```diff
@@ -29,6 +47,12 @@ beforeAll(async () => {
   svgImage = await fsPromises.readFile(
     path.join(process.cwd(), 'tests', '_test_inputs_', 'image.svg')
   );
+  pdfFile = await fsPromises.readFile(
+    path.join(process.cwd(), 'tests', '_test_inputs_', 'application.pdf')
+  );
+  mp3AudioFile = await fsPromises.readFile(
+    path.join(process.cwd(), 'tests', '_test_inputs_', 'audio.mp3')
+  );
```

Comment:
convert returned `Buffer` to `ArrayBuffer`

```suggestion
  pdfFile = nodeBufferToArrayBuffer(
    await fsPromises.readFile(
      path.join(process.cwd(), 'tests', '_test_inputs_', 'application.pdf')
    )
  );
  mp3AudioFile = nodeBufferToArrayBuffer(
    await fsPromises.readFile(
      path.join(process.cwd(), 'tests', '_test_inputs_', 'audio.mp3')
    )
  );
```